gooey/gui/util/quoting.py

```diff
@@ -1,4 +1,12 @@
-import re

```

Comment:
This import is still needed by `maybe_quote`, which is used by things in `widget_pack.py`